src/composables/types/parse.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  AgeRequestType,
+  ChoiceIdType,
+  ObservationType,
+  SexType,
+} from '@/composables/types';
+
+// TODO: find model in API for this type
```

Comment:
what about this todo? :)

Reply:
in documentation I can't find Model for this, so I have to ask accurate team

// IN PROGRESS 😉